chalk-solve/src/coherence.rs

```diff
@@ -11,11 +11,10 @@ use std::sync::Arc;
 pub mod orphan;
 mod solve;
 
-pub struct CoherenceSolver<'db, I: Interner, S, SC: Into<S> + Copy> {
+pub struct CoherenceSolver<'db, 'sb, I: Interner> {
```

Comment:
I think this (and `WfSolver`) could just have a single `'a` lifetime.

Reply:
Indeed! Done.